src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -463,13 +481,24 @@ private void updateCitizens()
     }
 
     /**
+     * Re-sorts the WorkOrders list according to the priorities inside the list.
+     */
+    private void sortWorkOrders()
+    {
+        workOrders.sort((first, second) ->
+            second.getPriority() > first.getPriority() ? 1 :
+                    (second.getPriority() < first.getPriority() ? -1 : 0)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3358)